x/nexus/keeper/transfer.go

```diff
@@ -148,6 +152,10 @@ func (k Keeper) EnqueueTransfer(ctx sdk.Context, senderChain exported.Chain, rec
 		asset = asset.Sub(fee)
 	}
 
+	if err := k.RateLimitTransfer(ctx, recipient.Chain.Name, asset, true); err != nil {
```

Comment:
the outgoing limit should be called after previous transfer merged

Reply:
The previous transfer (with status pending) was already rate limited when it was enqueued so we don't want to double count